stdlib/ballerina-http/src/main/java/org/ballerinalang/net/ws/WebSocketDispatcher.java

```diff
@@ -232,4 +245,21 @@ private static void pingAutomatically(WebSocketControlMessage controlMessage) {
             ErrorHandlerUtils.printError(ex);
         }
     }
+
+    private static void populateQueryParamMap(String payload, BMap<String, BString> params)
```

Comment:
There is a **HttpUtil** class method **getParamMap** which provides the same functionality. Since this is used in multiple places and also it is a part of URI can we add it to URIUtil class and make it available for both http and ws packages?